.clang-tidy

```diff
@@ -0,0 +1,16 @@
+Checks: '-*,readability-identifier-naming'
+CheckOptions:
+- key:             readability-identifier-naming.PrivateMemberPrefix
+  value:           'm_'
+- key:             readability-identifier-naming.ClassConstantCase
+  value:           aNy_CasE
+- key:             readability-identifier-naming.ClassConstantPrefix
+  value:           ''
+#- key:             readability-identifier-naming.PrivateMemberCase
+#  value:           CamelCase
+#- key:             readability-identifier-naming.FunctionCase
+#  value:           CamelCase
+#- key:             readability-identifier-naming.EnumCase
+#  value:           camelBack
+WarningsAsErrors: '*'
+#HeaderFilterRegex: '.+/cuberite/src/[^\.].+'
```

Comment:
Just uncomment, even if it isn't necessary for the script.
I would also suggest to change it to `/cuberite/src/\.?[^\.]`:
- Allows to have cuberite in `/cuberite` (e.g. root path)
- Allows `/cuberite/src/./Foo.cpp` and `/cuberite/src/x.cpp`